website/docs/d/nsxt_edgegateway_l2_vpn_tunnel.html.markdown

```diff
@@ -0,0 +1,67 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: vcd_nsxt_edgegateway_l2_vpn_tunnel"
+sidebar_current: "docs-vcd-datasource-nsxt-edgegateway-l2-vpn-tunnel"
+description: |-
+  Provides a datasource to read NSX-T Edge Gateway L2 VPN Tunnel sessions and their configurations.
+---
+
+# vcd\_nsxt\_edgegateway\_l2\_vpn\_tunnel
+
+Supported in provider *v3.11+* and VCD *10.2+* with NSX-T
```

Comment:
since provider 3.11 doesn't support VCD 10.2, we should probably say 10.4

Reply:
Updated